tests/fsharp/Compiler/CodeGen/EmittedIL/TaskGeneratedCode.fs

```diff
@@ -1104,7 +1104,7 @@ type Generic1InGeneric1<'T>() =
          extends [runtime]System.Object
   {
     .custom instance void [FSharp.Core]Microsoft.FSharp.Core.CompilationMappingAttribute::.ctor(valuetype [FSharp.Core]Microsoft.FSharp.Core.SourceConstructFlags) = ( 01 00 03 00 00 00 00 00 ) 
-    .class auto autochar sealed nested assembly beforefieldinit specialname clo@7<A>
+    .class auto autochar sealed nested assembly beforefieldinit specialname clo@7<T,A>
```

Comment:
This code is closure heavy, can we have it ilverified too?

Reply:
I think vlad may have added some ilverification, so I'll see what I can reuse.